### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by toabctl**
Describe a bit the current approach .

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -5,6 +5,62 @@ This repository contains kiwi_ definitions to build containers for
...
+dict/hash like data structures, the structures are merged where the last
+occurance wins).
+So we can have a chain of variable files like:
+
+- common/global.yml (used for *all* containers)
+- $image/common.yml (used for all $image containers)
+- $image/$distro.yml (used for the $distro specific container)
+- $image/$image.kiwi.j2 (the template to build $image for all distros)
+
+Theses are combined (eg. for a memcached SLE15SP1 image) with::
```
Comment (rhafer) [inline README.rst]: "The .kiwi output file (eg. for a memcached SLE15SP1 image) can be generated from the above files with::" ?

File: README.rst [hunk 2]
```diff
@@ -5,6 +5,62 @@ This repository contains kiwi_ definitions to build containers for
...
 
+Generating the .kiwi files from templates
+=========================================
+
+We want to build containers based on different base containers (eg. for
+openSUSE or SLE). To accomplish that, we use Jinja2 templates and
+YAML based (.yml) files which contain variables to fill the templates.
+
+The tool to combine the single template (usually .kiwi.j2) and the
+variable files(s) (usually .yml) is `j2gen`_
```
Comment (rhafer) [inline README.rst:16]: While not strictly wrong here I'd prefer to use "process" instead of "combine" here. But feel free to disagree :)

**Review by rhafer — APPROVED** [review]
Some minor nits. Feel free to ignore.

File: README.rst (hunk 1 above)
Comment (toabctl) [reply README.rst]: Done

File: README.rst (hunk 2 above)
Comment (toabctl) [reply README.rst:16]: Done

**Review by rhafer — APPROVED** [review]

**Pull request merged**